website_cms/templates/form.xml

```diff
@@ -127,4 +138,36 @@ We load each field explicitely here for now.
 
 </template>
 
+<template id="page_media" name="CMS Media Form">
+
+  <t t-call="website.layout">
+    <t t-if="False">
+      # we can override default action via form values
+    </t>
+
+    <t t-set="title" t-value="view.form_title" />
+
+    <div class="col-xs-12 col-md-10 col-md-offset-1">
+      <h2>
+        <span class="form_title" t-esc="view.form_title" />
+      </h2>
+
+      <ul class="nav nav-tabs nav-justified mt32">
+          <li role="presentation" class="active"><a t-att-href="website.cms_media_categ_links(main_object=main_object)">All</a></li>
+        <t t-foreach="media_categories" t-as="category">
+          <li role="presentation"><a t-att-href="website.cms_media_categ_links(main_object=main_object, category=category)" t-field="category.name"></a></li>
```

Comment:
category has a `icon` field where you can set the css class for it: can you add it here?
It should be something like `<i t-ignore="true" t-att-class="category.icon" />`